homeassistant/components/yale_smart_alarm/coordinator.py

```diff
@@ -55,17 +54,6 @@ async def _async_update_data(self) -> dict:
                     device["_state2"] = "unknown"
                     locks.append(device)
                     continue
-                if (
-                    lock_status
-                    and (
-                        "device_status.lock" in state or "device_status.unlock" in state
-                    )
-                    and jammed
-                ):
-                    device["_state"] = "jammed"
-                    device["_state2"] = "closed"
-                    locks.append(device)
-                    continue
```

Comment:
Whats going on here ?

Jammed is a valid state for the lock platform.

https://developers.home-assistant.io/docs/core/entity/lock?_highlight=jammed#properties

Reply:
I could not verify the returns from the api to trust it's jammed so I removed it again until I can do.